packages/legacy/core/App/utils/helpers.ts

```diff
@@ -996,6 +998,7 @@ export const connectFromScanOrDeeplink = async (
 
   // Try built in Credo methods first
   try {
+    agent.config.logger.info(`Attempting to connect from ${isDeepLink ? 'deep link' : 'scan'}, value: ${value}`)
```

Comment:
I think this should be resolved from DI now. Replace any other occurrences in the same file while you're at it. Examples in the code exists for copy.

Reply:
done